client/app/telecom/overTheBox/tasks/overTheBox-tasks.html

```diff
@@ -9,77 +9,21 @@
-                    <tbody data-ng-if="OverTheBoxTasks.loaders.init">
-                        <tr>
-                            <td colspan="3">
-                                <oui-spinner></oui-spinner>
-                            </td>
-                        </tr>
-                    </tbody>
-                    <tbody data-ng-if="OverTheBoxTasks.taskIds.length <= 0 && !OverTheBoxTasks.loaders.init">
-                        <tr>
-                            <td class="text-center"
-                                colspan="3"
-                                data-translate="overTheBox_no_task">
-                            </td>
-                        </tr>
-                    </tbody>
-                    <tbody data-ng-if="OverTheBoxTasks.taskIds.length > 0 && !OverTheBoxTasks.loaders.init">
-                        <tr data-ng-repeat="task in OverTheBoxTasks.tasks track by $index">
-                            <td data-title="{{:: 'overTheBox_tasks_name' | translate}}"
-                                data-ng-bind="task.name">
-                            </td>
-                            <td data-title="{{:: 'overTheBox_tasks_id' | translate}}"
-                                data-ng-bind="task.taskId">
-                            </td>
-                            <td class="text-right"
-                                data-title="{{:: 'overTheBox_tasks_status' | translate}}">
-                                <span class="label"
-                                      data-ng-class="{
-                                          'label-success': task.status === 'done',
-                                          'label-danger': tasks.status === 'error',
-                                          'label-info': tasks.status === 'doing',
-                                          'label-primary': tasks.status === 'todo'
-                                      }">
-                                    {{ 'overTheBox_tasks_status_' + task.status | translate }}
-                                </span>
-                            </td>
-                        </tr>
-                    </tbody>
-                </table>
-            </div>
-            <div class="mt-2"
-                 data-pagination-front
-                 data-go-to-page="false"
-                 data-items="OverTheBoxTasks.taskIds"
-                 data-paginated-items="OverTheBoxTasks.tasks"
-                 data-current-page="currentPage"
-                 data-nb-pages="maxSize"
-                 data-items-per-page="OverTheBoxTasks.filter.perPage"
-                 data-transform-item="OverTheBoxTasks.onTransformItem(item)"
-                 data-page-placeholder="{{ 'common_pagination_go_to_page' | translate }}"
-                 data-item-per-page-placeholder="{{'common_pagination_show_by_page' |translate }}">
-            </div>
+            <oui-datagrid rows="OverTheBoxTasks.tasks" row-loader="OverTheBoxTasks.onTransformItem($row)" >
```

Comment:
Have you test it??? you previously rename onTransformItem(...) to transformItem(...)

Be aware that using row-loader and oui-column -> sortable have some negative side effects, since you cannot order undefined data...